.eslintrc

```diff
@@ -12,6 +12,7 @@
   ],
   "rules": {
     "semi": 0,
+    "camelcase": ["error", {"properties": "never", "ignoreDestructuring": true}],
```

Comment:
I think when we approach adding things to the eslintrc, we should first try adding exceptions instead of turning off the rule right away
I mayyy change this when i merge my stuff since i have my own change this particular rule

Reply:
Definitely, camelcase isn't completely turned off here. These are exceptions for when we're using ids, like "user_id" in the database. Eslint wants ID to be capitalized.

Reply:
Definitely